en/04_Changelogs/5.2.0.md

```diff
@@ -100,6 +100,8 @@ This only applies to users created through the security section of the CMS. User
 
 ## API changes
 
+- The `$having` parameter in the [`Versioned::Versions()`](api:SilverStripe\Versioned\Versioned::Versions()) method has been deprecated. This parameter was never used, and has been removed from the method signature.
```

Comment:
Please add this under a `### silverstripe/versioned {#api-silverstripe-versioned}` heading, which should go after the framework one.

Reply:
Updated